src/web/page/web_page.rs

```diff
@@ -3,6 +3,7 @@ use iron::{headers::ContentType, response::Response, status::Status, IronResult,
 use serde::Serialize;
 use tera::Context;
 
+/// When making using a custom status, use a closure that coerces to a `fn(&Self) -> Status`
 #[macro_export]
```

Comment:
```suggestion
```
We're not using this from any other crate, so there's no need to export it.

Reply:
It's needed because of macro scoping rules

Reply:
It shouldn't be, you should `#[macro_use]` from above to get macros into context. `#[macro_export]` is _only_ for publicly exporting macros (and as a side-effect makes them visible from the root of the crate).